chemicalx/pipeline.py

```diff
@@ -70,6 +71,45 @@ def save(self, directory: Union[str, Path]) -> None:
         )
 
 
+def to_device(objects, device):
```

Comment:
I don't think this is necessary now that #84 is merged

Reply:
Agreed, will close the PR

Reply:
@GavEdwards I was only referring to the `to_device` function. There's still some benefit to consider adding accelerate for multi-GPU or TPU usage